src/relay/ir/pattern_functor.cc

```diff
@@ -18,8 +18,8 @@
  */
 
 /*!
- *  Copyright (c) 2018 by Contributors
- * \file src/tvm/relay/pattern_functor.cc
+ *  Copyright (c) 2019 by Contributors
```

Comment:
I think this is a typo - however @tqchen needs to comment as he was the last one who changed this. The first commit came in 2019, so you are safe here.